src/Cards/index.js

```diff
@@ -21,19 +21,37 @@ function selectRandomQuestions() {
 const currentRoundQuestions = selectRandomQuestions();
 const firstQuestion = triviaQuestions[currentRoundQuestions.pop()];
 
+function randomizedOptions(question) {
```

Comment:
I would name this `getRandomizedOptions` to signal that this is a function that is doing something. Otherwise this could be mistaken for a variable